src/rom.asm

```diff
@@ -228,6 +230,19 @@ not_mmc1:
   consider_mapper(11)
   consider_mapper(30)
   consider_mapper(31)
+// consider mapper 34 only with single CHR ROM page (BNROM)
+// when NINA-001 is supported this can be replaced with `consider_mapper(34)`
+  lli t2, 34
+  bne t0, t2,++
+  nop
+  ls_gp(lbu t2, chrrom_page_count)
+  bgtz t2,++
+  nop
++
```

Comment:
This extra `+` label should be removed, then the above branch targets can be changed to just `+`.